event-store/src/lib.rs

```diff
@@ -95,18 +92,11 @@ where
     }
 
     /// Query the backing store and return an entity `T`, reduced from queried events
-    fn aggregate<'b, E, T, A>(&'b self, query_args: A) -> Result<T, String>
+    fn aggregate<E, T, A>(&self, query_args: A) -> Result<T, String>
     where
-        E: Events + Send + Sync + 'b,
-        Q: 'b,
-        T: Aggregator<E, A, Q>
-            + Send
-            + Sync
-            + Serialize
-            + for<'de> Deserialize<'de>
-            + PartialEq
-            + 'b,
-        A: Clone + 'b,
+        E: Events + Send,
+        T: Aggregator<E, A, Q>,
```

Comment:
Note: This is far cleaner; the extra trait bounds are moved to the [`Aggregator` trait definition](https://github.com/repositive/event-store-rs/pull/54/files#diff-c55d6162c212e3644bc3beec892f8d8fR149).